README.md

```diff
@@ -32,6 +32,14 @@ Altenatively one can use the embedding API, see [src/mount.coffee](./src/mount.c
     rt.start (err, rt) ->
       # started
 
+With NoFlo 0.7.0, component discovery and caching 
```

Comment:
First sentence should be simplified as "Since NoFlo 0.7 fbp-manifest is used..."
We dont document pre 0.7 use anyway, so hinting about it is confusing ("begging the question")